packages/devtools_app/lib/src/screens/vm_developer/vm_object_model.dart

```diff
@@ -195,6 +195,15 @@ class LibraryObject extends VmObject {
 
   @override
   String? get name => obj.name;
+
+  late final String? vmName;
```

Comment:
Make this a getter on `obj` and remove the `initialize()` override below.

Reply:
Done